README.rst

```diff
@@ -73,6 +73,24 @@ Provides a ``GeometryField``, which is a subclass of Django Rest Framework
 geometry fields, providing custom ``to_native`` and ``from_native``
 methods for GeoJSON input/output.
 
+This field takes two optional arguments:
+
+``precision``: Passes coordinates through Python's builtin ``round()`` function (`docs
+<https://docs.python.org/3/library/functions.html#round>`_), rounding values to
+the provided level of precision. E.g. A Point with lat/lng of
+``[51.0486, -114.0708]`` passed through a ``GeometryField(precision=2)``
+would return a Point with a lat/lng of ``[51.04, -114.07]``.
```

Comment:
Shouldn't this be `51.05`?

Reply:
You would think, however as I learned when writing this PR, [5 rounds down in Python 3](https://stackoverflow.com/a/10825998/728583). I know, right?

Reply:
Actually, this is probably a better reference (see the "note") section: https://docs.python.org/dev/library/functions.html#round

Reply:
That is correct if the original value is 51.0456. This indeed rounds down to 51.04. But the digit in question is '8' so 51.0486 definitely rounds up to 51.05.

Reply:
Oh, yeah, right you are...

```py
In [1]: round(51.0486, 2)
Out[1]: 51.05
```

Will fix that, thanks.